ui/src/js/storyTelling-page-storyEdit.js

```diff
@@ -68,6 +68,34 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                 args: ["{that}.options.selectors.mainContainer", "{that}.options.selectors.pageContainer", "{that}.options.pageSetup.savingEnabled", "{that}.options.pageSetup.hiddenEditorClass"]
             }
         },
+        /*
+            TODO: Come up with a better name for this collection. Consider
+            making these values a configuration option in each block grade,
+            maybe even making the block contentString model relays based on it
+        */
+        requiredBlockValues: {
+            "text": ["heading", "text", "simplifiedText"],
+            "image": ["imageUrl"],
+            "audio": ["mediaUrl"],
+            "video": ["mediaUrl"]
+        },
+        modelRelay: {
```

Comment:
It might be possible to combine these 2 model relays into one by:
1. Call `sjrk.storyTelling.page.storyEdit.removeEmptyBlocks` in the "singleTransform" of "editorStoryToPreviewer" relay;
2. Get rid of "excludeEmptyBlocks" relay.